packages/git-raw-commits/index.js

```diff
@@ -52,38 +52,31 @@ function gitRawCommits (rawGitOpts, rawExecOpts) {
     gitOpts.debug('Your git-log command is:\ngit ' + args.join(' '))
   }
 
-  var isError = false
-
   var child = execFile('git', args, {
     cwd: execOpts.cwd,
     maxBuffer: Infinity
+  }, function (err) {
+    if (err != null) {
+      readable.emit('error', err)
+    }
+
+    readable.emit('close')
```

Comment:
Error checking is changed from checking for output on STDERR to checking the error in a callback function.  `error` is emitted if a non-null error is passed to the callback, and `close` is always emitted.